ontology/uco/observable/observable.ttl

```diff
@@ -12905,13 +12938,6 @@ observable:statusesCount
 	rdfs:range xsd:nonNegativeInteger ;
 	.
 
-observable:storageCapacityInBytes
```

Comment:
This needs to be restored for now.  Please add `owl:deprecated "true"^^xsd:boolean` to the restored property definition.

As part of the discussion for [Issue 493](https://github.com/ucoProject/UCO/issues/493) on Nov. 17, we're going to decide whether to delete the property in UCO 2.0.0, or leave it as a stub concept for some long time.  So, on or after Nov. 17, we're going to have one additional patch to update the `rdfs:comment`.

Reply:
`owl:deprecated` is described in [this section of the OWL 2 syntax document](https://www.w3.org/TR/owl2-syntax/#Annotation_Properties).

Reply:
Turned out I was a little off on the deprecation syntax.  From [this document](https://www.w3.org/TR/2012/REC-owl2-mapping-to-rdf-20121211/), Table 16, 4th row from the bottom, the property is supposed to be given the type `owl:DeprecatedProperty`.

Patch pushed.